back/src/forms/compat.ts

```diff
@@ -11,9 +11,9 @@ import { Bsdd } from "./types";
  * @returns
  */
 export function simpleFormToBsdd(
-  form: Form & { transportSegments?: TransportSegment[] }
+  form: Form & { transporters?: BsddTransporter[] }
 ): Bsdd {
-  const [transporter2, transporter3] = form.transportSegments ?? [];
+  const [transporter, transporter2, transporter3] = form.transporters ?? [];
```

Comment:
ca marche ca ? ils sont dans l'ordre ? ca serait top d'avoir une garantie la dessus mais je ne sais pas si c'est possible

Reply:
En fait le typage n'est pas mieux inféré comme ça. Cf https://www.executeprogram.com/courses/typescript-basics/lessons/undefined-in-arrays. On pourrait activer éventuellement `noUncheckedIndexedAccess` si on voulait. Pour l'heure je vais me contenter d'ajouter des `?`.